bookshelf/src/CloudsqlModel.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Samples\Bookshelf;
+
+use PDO;
+
+/**
+ * Class CloudsqlModel implements the DataModel with a mysql database.
+ *
+ * Set the three environment variables MYSQL_DSN, MYSQL_USER,
+ * and MYSQL_PASSWORD.
+ */
+class CloudsqlModel implements DataModel
+{
+    /**
+     * Creates a new PDO instance and sets error mode to exception.
+     *
+     * @return PDO
+     */
+    private function newConnection()
+    {
+        $pdo = new PDO(getenv('MYSQL_DSN'), getenv('MYSQL_USER'),
```

Comment:
When running tests on Travis, Jenkins, etc., it's very useful to have these in environment variables, so that there are no conflicts when merging between a base and a fork.